### PR TITLE
[DOC] Document supported versions of Java, Python, Scala

### PR DESCRIPTION
**Pull request by felixcheung**
related to #151

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -55,9 +55,11 @@ GraphFrames depends on Apache Spark, which is available for download from the
 GraphFrames should be compatible with any platform which runs Spark.
 Refer to the [Apache Spark documentation](http://spark.apache.org/docs/latest) for more information.
 
-GraphFrames is compatible with Spark 1.4+.  However, later versions of Spark include major improvements
+GraphFrames is compatible with Spark 1.6+.  However, later versions of Spark include major improvements
 to DataFrames, so GraphFrames may be more efficient when running on more recent Spark versions.
 
+GraphFrames is tested with Java 7, Python 2 and 3, and running against Spark 1.6+ (Scala 2.10.6) and Spark 2.0+ (Scala 2.11.8).
```
Comment (thunterdb) [inline docs/index.md]: Can you drop the minors of the scala versions?

Comment (felixcheung) [inline docs/index.md]: done - I think you mean patch part of the scala versions

**Comment by felixcheung** [comment]
@thunterdb when you get a chance. I'll open a PR for #166 too once this is in. (although, we probably need to snap version back to 0.3 manually instead of 0.4.0-SNAPSHOT)

**Comment by jkbradley** [comment]
LGTM, thank you!

**Pull request merged**